charts/airbyte/values.yaml

```diff
@@ -322,7 +419,7 @@ server:
   image:
     repository: airbyte/server
     pullPolicy: IfNotPresent
-    tag: 0.39.37-alpha
+    tag: 0.39.36-alpha
```

Comment:
oh wait it looks like some of the versions are not right in this file specifically

Reply:
Fixed